c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -3,6 +3,9 @@ goog.provide('app.viewDetailsDirective');
 
 goog.require('app');
 
+/** @suppress {extraRequire} */
```

Comment:
Why do you need this ``@suppress``?
Perhaps you don't need it if you fixed the ``@param`` issue below?